xml/System/String.xml

```diff
@@ -12164,7 +12217,7 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
   
  The following example uses the <xref:System.String.Substring%2A> method to separate key/value pairs that are delimited by an equals ("=") character.  
```

Comment:
the sample before this line seems that it would work with Try .NET as well and then the note could say all since it seems all samples would be runnable

Reply:
Done.